torchaudio/pipelines/_wav2vec2/utils.py

```diff
@@ -67,6 +67,38 @@ def _get_de_labels():
     )
 
 
+def _get_vp_en_labels():
+    return (
+        "|",
+        "e",
+        "t",
+        "o",
+        "i",
+        "a",
+        "n",
+        "s",
+        "r",
+        "h",
+        "l",
+        "d",
+        "c",
+        "u",
+        "m",
+        "p",
+        "f",
+        "g",
+        "w",
+        "y",
+        "b",
+        "v",
+        "k",
+        "x",
+        "j",
+        "q",
+        "z",
+    )
```

Comment:
wondering if the difference from ``_get_en_labels`` is just the lower case? Maybe we can define a standard label method so that the outputs of all english models can be transformed to that standard labels (e.g. in an alphabetic order in lower case).

Reply:
The order is different. `E T A O N I ...` vs `e t o i a n ...`.
That's an interesting idea, but to do that, we need to re-order the pertained weights on the final layer, which is kinda complicated and I wonder if it's really worth it.

Reply:
Instead of manipulating the model weight, we can build a dictionary that maps each model's labels to a standard one. But that takes the same lengths of code though..

Reply:
I think that is something client code can do on top of the existing API. I do not think that we have a strong reason that we have to do that, or the value propositions of such API.

Reply:
Yep, that makes sense.